app/bundles/PageBundle/EventListener/ReportSubscriber.php

```diff
@@ -524,8 +524,8 @@ public function onReportGraphGenerate(ReportGraphEvent $event)
                 case 'mautic.page.graph.pie.devices':
                     $queryBuilder->select('ds.device, COUNT(distinct(ph.id)) as the_count')
                         ->groupBy('ds.device');
-                    $data  = $queryBuilder->execute()->fetchAll();
-                    $chart = new PieChart();
+                    $data     = $queryBuilder->execute()->fetchAllAssociative();
```

Comment:
Remove extra spaces here.

Reply:
Fixed in commit 1387c7eb8479a51b9db212aa5ead281c9b9c7164